### ISSUE TITLE
Fullscreen is not possible with LOVE 11.3 on Android (?)

### ISSUE THREAD

**Issue by slime73** [love2d/love#1533]
**[Original report](https://bitbucket.org/rude/love/issue/1528) by Michael Franz (Bitbucket: [mishafranz](https://bitbucket.org/mishafranz), ).**

----------------------------------------

Hello,

I am trying to get Fullscreen on Android with 11.3 without luck.

In both portrait and landscape modes there are unused zones on the bottom and sometimes on a side of the screen.

Here is how this looks:

![](https://bitbucket.org/repo/A6ddy/images/2802792955-FullScreen-Android10-Portrait.png)
‌

![](https://bitbucket.org/repo/A6ddy/images/1425127195-FullScreen-Android10-Landscape.png)
‌

![](https://bitbucket.org/repo/A6ddy/images/548055880-FullScreen-Android7-Portrait.png)
‌

![](https://bitbucket.org/repo/A6ddy/images/2382276102-FullScreen-Android7-Landscape.png)
I am using 11.3 embed for this test.

Tested on Android 10 \(2 first pics\) and Android 7 \(2 last pics\).

On Windows fullscreen seems to work as expected.

Here are the test scripts:

conf.lua:

‌

```
function love.conf(t)
  t.version = "11.3"

  t.window.title = "Full Screen Test"
  t.window.borderless = true
  t.window.fullscreen = true
  -- t.window.fullscreentype = "exclusive"
  t.window.highdpi = false

  t.modules.joystick = false
  t.modules.mouse = false 
  t.modules.physics = false
  t.modules.audio = false
  t.modules.sound = false
  t.modules.video = false
  t.modules.image = false

  t.accelerometerjoystick = false
  t.externalstorage = false 
end
```

‌

main.lua:

‌

```
local x, y, w, h

function love.load()
  if arg[#arg] == "-debug" then require("mobdebug").start() end

  x = 0
  y = 0
  -- w, h = love.graphics.getDimensions()

  x, y, w, h = love.window.getSafeArea( )
end

function love.draw()
  love.graphics.setColor(1, 1, 1)
  love.graphics.rectangle("line", x, y, w, h)
end
```

‌

Manifest \(I only change landscape/portrait\):

‌

```
<?xml version="1.0" encoding="utf-8"?>
<manifest package="org.love2d.android.executable"
  android:installLocation="auto" xmlns:android="http://schemas.android.com/apk/res/android">

  <!-- OpenGL ES 2.0 -->
  <uses-feature android:glEsVersion="0x00020000" />

  <application
      android:allowBackup="true"
      android:icon="@drawable/love"
      android:label="LÖVE for Android"
      android:theme="@android:style/Theme.NoTitleBar.Fullscreen" >
      <activity
        android:name="org.love2d.android.GameActivity"
        android:configChanges="orientation|screenSize|keyboard|keyboardHidden|navigation"
        android:label="LÖVE for Android"
        android:launchMode="singleTop"
        android:screenOrientation="landscape" >
        <intent-filter>
          <action android:name="android.intent.action.MAIN" />
          <category android:name="android.intent.category.LAUNCHER" />
          <category android:name="tv.ouya.intent.category.GAME" />
        </intent-filter>
        <intent-filter>
          <action android:name="android.intent.action.VIEW" />
          <category android:name="android.intent.category.DEFAULT" />
          <category android:name="android.intent.category.BROWSABLE" />
          <data android:scheme="file" />
          <data android:mimeType="*/*" />
          <data android:pathPattern=".*\\.love" />
          <data android:host="*" />
        </intent-filter>
      </activity>
    </application>
</manifest> 
```

‌

Any ideas?

‌

Thanks,

Michael.

**Comment by MikuAuahDark** [comment]
I updated SDL to 2.0.12 and fixed some conflicting code. Can you confirm if this is still the case for love-android `master` branch?

**Comment by MikuAuahDark** [comment]
I re-tested this in Redmi Note 9 Pro running Android 10 and looks like (and I believe) this has been fixed in love2d/love-android@31d7affd91cd5ab021b2058fd2c06b1ed593e261. Also see this forum posts https://love2d.org/forums/viewtopic.php?f=4&t=87576&start=10
